test/tests/ver-1a/ver-1a.i

```diff
@@ -1,21 +1,49 @@
-temperature = 2.373e3
-initial_pressure = 1e6
-kb = 1.38e-23
-length_unit = 1e6 # number of length units in a meter
-pressure_unit = 1 # number of pressure units in a Pascal
+# Verification Problem #1a from TMAP4/TMAP7 V&V document
+# Tritium diffusion through SiC layer with depleting source at 2100 C.
+# No Sorret effect, solubility, or trapping included.
+
+# Physical Constants note that we do NOT use the same number of digits as in TMAP4/TMAP7.
+# This is to be consistent with PhysicalConstant.h
+kb = 1.380649e-23 # Boltzmann constant J/K
+R = 8.31446261815324 # Gas constant J/mol/K
+
+# Data used in TMAP4/TMAP7 case
+length_unit = 1e6 # conversion from meters to microns
+temperature = 2373 # K
+initial_pressure = 1e6 # Pa
+volume_enclosure = '${fparse 5.20e-11*length_unit^3}' # microns^3
```

Comment:
Good to know that we can use file variables with `fparse` expression for substitution in the file! Never tried this one.

Reply:
It's a life changer! I use it all the time now! Being able to do all this at the top of the input file increases clarity, and is especially useful when combining input files.